src/entities/Committee/utils.ts

```diff
@@ -0,0 +1,39 @@
+import { Committee, CommitteeName, DclData } from '../../clients/DclData'
+
+export async function getCommitteesWithOpenSlots(): Promise<Committee[]> {
+  // const { teams: committees } = await DclData.get().getData()
```

Comment:
Reminder to remove this mock

Reply:
We should try/catch somewhere because the DclData call could fail. 
One implementation path could be to try/catch here, and return an empty list. But this would mean we'd have to try catch on other places that we use `getData()` too.

Maybe `getData()` could return a valid `null object` that has `{ teams: [] }` and any other expected data when the request fails.